### ISSUE TITLE
Liking of city chat edits not registering

### ISSUE THREAD

**Issue by Scott176** [Team357/cityonchain#5]
Liking a post in city chat is processed correctly by MoneyButton. However, it results in a pulsing heart symbol and the like doesn't increment the like counter.

**Comment by dyfqpl1** [comment]
if results in a pulsing heart,it may request moneyButton;at sometimes,moneybutton response lately

**Comment by WaterEverySeed** [comment]
> if results in a pulsing heart,it may request moneyButton;at sometimes,moneybutton response lately

Hello @dyfqpl1 The like payment is going through on moneybutton, but is not being registered on the like counter on cityonchain.

**Comment by Scott176** [comment]
> if results in a pulsing heart,it may request moneyButton;at sometimes,moneybutton response lately

No, that isn't right. The MoneyButton payment is processed fine. I still get the pulsing heart after the payment has gone through. When I reload the page the like counter has not registered.

**Comment by dyfqpl1** [comment]
which city? I have a try

**Comment by Scott176** [comment]
> which city? I have a try

Any city. But for the latest one I tested on was Colombo.

**Comment by dyfqpl1** [comment]
![图片](https://user-images.githubusercontent.com/37652341/75408907-20c20d00-5952-11ea-960d-51c6c75ae1a4.png)
I hava a try, it's ok

**Comment by Scott176** [comment]
> ![图片](https://user-images.githubusercontent.com/37652341/75408907-20c20d00-5952-11ea-960d-51c6c75ae1a4.png)
> I hava a try, it's ok

Nonetheless, it isn't working for me.  And if it's not working for me then no doubt others are experiencing the same issue.

**Comment by dyfqpl1** [comment]
ok,I will let others try,thanks a lot for your issue

**Comment by Scott176** [comment]
> ok,I will let others try,thanks a lot for your issue

Thanks for your attention to this problem. Just so you know, the problem first started a few days or so ago. Prior to that it has been working fine.

**Comment by Scott176** [comment]
I tried logging out of MoneyButton and back in, but that made no difference. This is still an issue with some users.
